### PR TITLE
Fixed #39 From name is always empty

### LINKED ISSUE #39: From name is always empty
In previous versions, you could just do `from_email="Company <info@company.com>"`.

However, as of https://github.com/elbuo8/sendgrid-django/commit/9cb266812d6beaf063fa26359f12ebc3f71f4b8d, only the e-mail is passed to the new Email() method of the sendgrid package.

This means the From name is now always empty.

### PR DESCRIPTION
**Pull request by andriisoldatenko**
## Fix #39 

Related issues:
- https://github.com/sendgrid/sendgrid-python/issues/277
- RFC: https://tools.ietf.org/html/rfc2822#section-3.4
- Django docs: https://docs.djangoproject.com/en/1.10/topics/email/#emailmessage-objects

After PR merged:
image

### CODE REVIEW

**Comment by andriisoldatenko** [comment]
@elbuo8 Please review
cc @thinkingserious @GlennV

**Comment by GlennV** [comment]
lgtm

File: sgbackend/mail.py [hunk 1]
```diff
@@ -66,7 +71,12 @@ def send_messages(self, emails):
 
     def _build_sg_mail(self, email):
         mail = Mail()
-        mail.set_from(Email(email.from_email))
+        from_name, from_email = rfc822.parseaddr(email.from_email)
+        # Python sendgrid client should improve
+        # sendgrid/helpers/mail/mail.py:164
```
Comment (elbuo8) [inline sgbackend/mail.py]: @thinkingserious

Comment (sendgrid-gists) [inline sgbackend/mail.py]: Nice! We will track here: https://github.com/sendgrid/sendgrid-python/issues/277

**Pull request merged**